src/extensions/b2b/index.md

```diff
@@ -14,8 +14,11 @@ For {{site.data.var.ece}} projects, see [Set up Magento B2B module]({{ site.base
 
 1. Change to your Magento installation directory and enter the following command to update your `composer.json` file and install the {{site.data.var.b2b}} extension:
 
+{:.bs-callout-info}
+Compatible Magento - B2B version must be provided in the command.
```

Comment:
```suggestion
Compatible Magento B2B version must be provided in the command.
```

Reply:
@BarnyShergold Updated the changes. Thanks.